### DISCUSSION TITLE
How to load admin and user panel without anti filter? چه جوری بدون فیلتر شکن وارد پنل ادمین و پنل کاربری بشیم

### DISCUSSION THREAD

**Discussion by FarshadFaramarzlou** [hiddify/Hiddify-Manager#146] [Q&A سوالات رایج] [answered]
چه جوری بدون فیلتر شکن وارد پنل ادمین و پنل کاربری بشیم؟

**Comment by bestboy2022** [comment, marked as answer]
در حالتی که ip سرور شما فیلتر نباشد (یا به اصطلاح  ipکثیف نباشد) بدون هیچ فیلترشکنی میتوان وارد پنل ادمین شد. اما زمانی که ipسرور کثیف باشد برای اینکه بتوانید بدون فیلترشکن وارد پنل کاربری خود شوید کافیست یک دامنه در حالت cdn (ابر پروکسی در کلاد فلر روشن باشد) به سرور متصل کرده، و دامنه را در پنل هایدیفای ثبت کرده بدین ترتیب بعد از ثبت دامنه cdn، در پنل شما یک لینک دیگر برای ورود به پنل ادمین برای شما اضافه خواهد شد که میتوان از طریق آن بدون فیلترشکن وارد پنل ادمین شد.

![IMG_20230215_150909_000](https://user-images.githubusercontent.com/124868504/219017611-4d438fce-d004-4205-a98f-a9276bc0b642.jpg)
![IMG_20230215_025501_229](https://user-images.githubusercontent.com/124868504/219017630-207e05dc-5851-4ef1-b0a1-2a92839abd3c.jpg)
![IMG_20230215_025505_265](https://user-images.githubusercontent.com/124868504/219017663-752e952b-6021-4724-8baf-9dccfeace8b3.jpg)
![IMG_20230215_025508_822](https://user-images.githubusercontent.com/124868504/219017668-7fc69be9-2ca3-45c5-9ed6-45779914f56b.jpg)

**Reply by FarshadFaramarzlou** [reply]
[
![image](https://user-images.githubusercontent.com/12745228/218964224-26322d48-fc70-4cfd-b908-b8191517c266.png)
](url)

کارهایی که گفتین انجام شد، نتیجه خطای بالا هستش

![Untitled](https://user-images.githubusercontent.com/12745228/218966367-aeffebf0-947d-48a1-a351-4b077a868fea.png)


اختلافی که توی مراحل کار دیدم اخطار های cloudflare بود.

لطفا راهنمایی کنید چطور این مشکل رو حل کنم؟

**Reply by bestboy2022** [reply]
طبق پیغام کلاد فلر در کادر زرد رنگ بالایی، دامین شما به کلاد فلر متصل نیستش. برای اتصال دامنه تون به کلاد فلر باید وارد پنل دامنه خودتون بشید و نیم سرور های اون رو به نیم سرور های کلاد فلر تغییر بدید. (در گوگل "نحوه اتصال دامنه به cloudflare" رو سرچ کنید)

**Reply by FarshadFaramarzlou** [reply]
بله ممنون، با دامنه دیگه ای که از freenom.com دارم انجام دادم اما متاسفانه دوباره نا موفق بودم

![image](https://user-images.githubusercontent.com/12745228/219008039-4dd7d8e0-c3f3-47f5-aa23-7ab794326d8e.png)
تصویر کلودفلیر
![image](https://user-images.githubusercontent.com/12745228/219008826-2603e8bf-859f-470a-84b7-569a5836b87d.png)


وقتی آدرس دامنه رو با فیلترشکن میزنم میاره! ولی بدون فیلتر شکر میزنم نمیاره و خطای پایین میاد

![image](https://user-images.githubusercontent.com/12745228/219009750-f2f41f21-b49e-4645-adf3-8565a89e26c4.png)

**Reply by bestboy2022** [reply]
اگه تمام مراحل رو درست رفته باشید و همچنان با فیلترشکن وارد میشید، احتمالا دامنه شما فیلتره از یک دامنه دیگه استفاده کنید و مجدد تست کنید.(ترجیحا از دامنه های رایگان استفاده نکنید)
